src/core/etl/src/Flow/ETL/DSL/functions.php

```diff
@@ -349,7 +348,7 @@ function to_branch(ScalarFunction $condition, Loader $loader) : Loader\Branching
 }
 
 #[DocumentationDSL(module: Module::CORE, type: DSLType::TRANSFORMER)]
-function rename_style(Style $style) : Transformer\Rename\RenameCaseEntryStrategy
+function rename_style(StringStyles $style) : Transformer\Rename\RenameCaseEntryStrategy
```

Comment:
It's a hard BC break, it also creates a coupling between Scalar Function and Rename Transformer. 

I would mve that Stringstyles class to `Flow\ETL\String\StringStyles` namespace first keeping those two others and making them deprecated. 
The I would do a union type here and in that function `Style|StringStyles` and mark old classes as deprecated which I will remove before 1.0 release

Reply:
We're lucky that I noticed this before we did `0.16`, so there is no BC in the matter of `Style` class, so IMO we need to decide how to merge them (on what I'm open).

Reply:
ah wait, right, this is a new method that you just added 🤦‍♂️ so I would just move that StringStyles to `Flow\ETL\String\StringStyles` and make sure that the scalar function can use it and deprecated that other one used currently in ScalarFunction